k9mail/src/main/java/com/fsck/k9/ui/settings/SettingsActivity.kt

```diff
@@ -16,8 +15,11 @@ import com.xwray.groupie.Item
 import com.xwray.groupie.Section
 import com.xwray.groupie.kotlinandroidextensions.ViewHolder
 import kotlinx.android.synthetic.main.activity_settings.*
+import org.koin.android.ext.android.inject
 
 class SettingsActivity : K9Activity() {
+    private val viewModel: SettingsViewModel by inject()
```

Comment:
is there a reason you didn't use koin's `by viewModel` here?

Reply:
Good catch. I thought I did use `by viewModel` :)